aten/src/ATen/core/dispatch/DispatchTable.h

```diff
@@ -100,7 +100,18 @@ class ThreadsafeOperatorTable_ final {
    }
 
    static std::string dispatch_key_to_string(TensorTypeId id) {
-     return std::string(toString(tensorTypeIdToBackend(id))) + "[" + toString(id) + "]";
+      // TODO Find better way to stringify tensor type ids without relying on backend
```

Comment:
C10_DECLARE_TENSOR_TYPE can just store the passed name in the member field or something.

Reply:
yes, that's what I had in mind